tools/ClassGen/Backends/OpenCL/OpenCLSpecificInstrs.h

```diff
@@ -23,7 +23,7 @@ BB.newBackendSpecificInstr("OCLConvolution")
     .addOperand("Bias", OperandKind::In)
     .addMember(MemberType::SizeT, "Kernel")
     .addMember(MemberType::SizeT, "Stride")
-    .addMember(MemberType::SizeT, "Pad")
+    .addMember(MemberType::VectorSizeT, "Pads")
```

Comment:
This is OK. 

BWT,  I noticed that no Group parameter was added to OpenCL convolutions, when we added group convolutions support. May be we should add this parameter for the sake of completeness. In a different PR, of course.

Reply:
Will work on group parameter.